classes/context/Context.inc.php

```diff
@@ -68,6 +68,14 @@ public function isDoiTypeEnabled(string $doiType): bool
         return in_array($doiType, $this->getData(Context::SETTING_ENABLED_DOI_TYPES));
     }
 
+    /**
+     * Retrieves array of enabled DOI types (items are one of Repo::doi()::TYPE_*).
+     */
+    public function getEnabledDoiTypes(): ?array
```

Comment:
One of the benefits of writing a "getter method" is to be able to use type hinting. It's no big deal, but I might be inclined to return an empty array if no doi types are enabled:

```php
return $this->getData(...) ?? [];
```

That way you get to remove the null possibility from the return type, and the "consumer" of the method doesn't have to perform any type checks.